scripts/backport.js

```diff
@@ -0,0 +1,117 @@
+require(`dotenv`).config()
+
+const yargs = require(`yargs`)
+const { Octokit } = require(`@octokit/rest`)
+const childProcess = require(`child_process`)
+
+if (!process.env.GITHUB_ACCESS_TOKEN) {
+  throw new Error(`GITHUB_ACCESS_TOKEN env var not set`)
+}
+
+const octokit = new Octokit({
+  auth: `token ${process.env.GITHUB_ACCESS_TOKEN}`,
+})
+
+const argv = yargs
+  .command(
+    `* <release> [pr]`,
+    `Backports merged PR into release branch and open Cherry PR`,
+    commandBuilder =>
+      commandBuilder
+        .positional(`release`, { type: `string` })
+        .positional(`pr`, { type: `number` })
+  )
+  .check(argv => {
+    if (!/^3\.\d+$/.test(argv.release)) {
+      throw new Error(`"${argv.release}" is not a release version`)
+    }
+
+    if (!argv.pr) {
+      throw new Error(`PR number is required`)
+    }
+
+    return true
+  }).argv
+
+const repo = `gatsby`
+const owner = `gatsbyjs`
+
+// no try/catches - if it fails at any point - let it fail and just do stuff manually ;)
+// this is just for happy path - if there is extra work needed - this script won't be able to do it
+async function run() {
+  const result = await octokit.pulls.get({
+    owner,
+    repo,
+    pull_number: argv.pr,
+  })
+
+  if (!result.data.merged_at) {
+    throw new Error(`That pull request was not merged`)
+  }
+
+  const commitsha = result.data.merge_commit_sha
+
+  const commitMeta = await octokit.git.getCommit({
+    owner,
+    repo,
+    commit_sha: commitsha,
+  })
+
+  // get first line
+  const commitMessage = commitMeta.data.message.split(`\n`)[0]
+
+  if (!commitsha) {
+    throw new Error(`Can't get merge commit sha`)
+  }
```

Comment:
I just realized we try to use `commitsha` (lines 54-58) before checking if it's actually available ;) (seems like we never hit throwing case, but if we have it let's make it correct at least (move check before `getCommit` call)
```suggestion
  const commitsha = result.data.merge_commit_sha
  
  if (!commitsha) {
    throw new Error(`Can't get merge commit sha`)
  }

  const commitMeta = await octokit.git.getCommit({
    owner,
    repo,
    commit_sha: commitsha,
  })

  // get first line
  const commitMessage = commitMeta.data.message.split(`\n`)[0]
```